packages/instantsearch.js/src/lib/utils/__tests__/hydrateRecommendCache-test.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  createSearchClient,
+  createSingleSearchResponse,
+} from '@instantsearch/mocks';
+import algoliaSearchHelper from 'algoliasearch-helper';
+
+import { hydrateRecommendCache } from '../hydrateRecommendCache';
+
+describe('hydrateRecommendCache', () => {
+  it('should hydrate the helper with the recommend cache', () => {
```

Comment:
add maybe a test with a broken response?

Reply:
What do you mean by broken response ? I made one with an undefined `recommendResults` which could happen if they don't have recommend widgets